includes/Modules/Analytics.php

```diff
@@ -121,6 +121,13 @@ public function register() {
 		add_action( 'template_redirect', $this->get_method_proxy( 'register_tag' ) );
 
 		add_filter( 'googlesitekit_proxy_setup_url_params', $this->get_method_proxy( 'update_proxy_setup_mode' ) );
```

Comment:
This filter call (and also the entire `googlesitekit_proxy_setup_url_params` filter being applied as well) should be removed, per the ACs.

Reply:
Removed.

Reply:
You've now removed the filter _usage_ here, can you also remove the entire filter itself please? The filter hook shouldn't even exist anymore.

Reply:
Ok, got it. Removed the filter too.